package.json

```diff
@@ -145,4 +145,4 @@
     "trailingComma": "es5",
     "printWidth": 120
   }
-}
+}
```

Comment:
Do we care about this? https://stackoverflow.com/a/729795

Reply:
I think the style guide says we should have line breaks at the end, but it seems our auto formatting is fighting that...